### ISSUE TITLE
Ctrl + Clkcing the Reference style linking will cause KIO to open a file  instead of opening the link.

### ISSUE THREAD

**Issue by v3ss0n** [pbek/QOwnNotes#1953]
#### Expected behaviour

CTRL+ Clicking the link should open the http link in the browser

#### Actual behaviour

![image](https://user-images.githubusercontent.com/419606/99454644-18cd5180-2955-11eb-8d4e-14ba1a2415e8.png)

#### Steps to reproduce

- Make a note 
- paste below code
```markdown
[You can use numbers for reference-style link definitions][1]
[1]: https://www.qownnotes.org
```
- Control click the link on the first line 

Only happesn on the refrence style links , other links works fine.

**Comment by pbek** [comment]
Yes, reference links never were implemented for `Ctrl + Click` because of the complexity if them.

**Comment by v3ss0n** [comment]
Aren't they the same with Normal Links? What kind of complexity ?

**Comment by pbek** [comment]
You have to check the whole note for a suitable link that should not be in a code block, comment and so on.

**Comment by v3ss0n** [comment]
Wow , i didn't realize that , just realized now. that gonna be so bad in terms of complexity and performance.
Since they are not in any kind of scheme at all. Should we raise it to commonmark spec guys ?

**Comment by v3ss0n** [comment]
This is how they do in vscode : https://github.com/microsoft/vscode/pull/75357/commits/01f7276b7eb02fe07802e66bc44ba9a3d8ab35ce

**Comment by pbek** [comment]
That already works now.
